melvs/melvsboot/src/test/java/com/melvs/melvsboot/MelvsbootApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.melvs.melvsboot;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class MelvsbootApplicationTests {
+
+	@Test
+	void contextLoads() {
+	}
```

Comment:
probably a good idea to put an actual test here, I would recommend adding actuations at least. I'm guilty of this too with my project, but there is reference code for tests over at the base folder. We should implement actual tests too when we are able to.

Reply:
I would like to have coding to be TDD (Test driven development) where development starts with the creation of test scripts then building the logic around it. I am still doing refresh to add more into it.